answers/views.py

```diff
@@ -77,23 +77,29 @@ def get(self, request, hunt_pk):
         }
         return render(request, 'queue.html', context)
 
+    # TODO(erwa): Move this method out of AnswerView, as this is being used from
+    # puzzles/views.py as well
     @staticmethod
-    def __update_slack_with_puzzle_status(answer, status):
+    def update_slack_with_puzzle_status(answer, status):
         slack_client = SlackClient.getInstance()
         puzzle_channel = answer.puzzle.channel
-        if status == Answer.PARTIAL:
-            slack_client.send_message(puzzle_channel,
-                                      "%s is PARTIALLY CORRECT!" %
-                                      answer.text.upper())
-        elif status == Answer.INCORRECT or status == Answer.CORRECT:
-            slack_client.send_message(puzzle_channel, "%s is %s!" %
-                                                      (answer.text.upper(), status))
-
+        message = ''
+        if status == Answer.NEW:
+            message = ("NEW answer '%s' has been guessed for puzzle '%s'." %
+                      (answer.text, answer.puzzle.name))
+        elif status == Answer.SUBMITTED:
+            message = ("'%s' has been SUBMITTED for puzzle '%s' on the hunt website." %
+                      (answer.text, answer.puzzle.name))
+        elif status in [Answer.PARTIAL, Answer.INCORRECT, Answer.CORRECT]:
```

Comment:
else? are there any other statuses?

Reply:
No other statuses. Maybe I’ll add an `else “UNEXPECTED STATUS!”` to avoid forgetting to update this method if we ever add a new status

Reply:
Added an else case:
```
        else:
            message = ("Unexpected status '%s' for answer '%s' for puzzle '%s'" %
                      (status, answer.text, answer.puzzle.name))
```

Reply:
Probably should terminate early here.

Reply:
Changed to `logger.error()` and added a `return`.